zqd/handlers_test.go

```diff
@@ -290,19 +291,27 @@ func newCoreAtDir(t *testing.T, dir string) (*zqd.Core, *api.Connection, func())
 	}
 }
 
-// postSpaceData POSTs the provided data source as a log in to the provided space
-// directory.
-func postSpaceData(t *testing.T, c *api.Connection, spaceName, data string) *api.Stream {
+func writeTempFile(t *testing.T, contents string) string {
 	pattern := strings.ReplaceAll(t.Name(), "/", "-")
 	f, err := ioutil.TempFile("", pattern)
 	require.NoError(t, err)
-	name := f.Name()
 	defer f.Close()
-	defer os.Remove(name)
-	_, err = f.WriteString(data)
+	_, err = f.WriteString(contents)
 	require.NoError(t, err)
+	return f.Name()
+}
+
+// postSpaceLogs POSTs the provided strings as logs in to the provided space, and returns a slice of any payloads that the server sent.
+func postSpaceLogs(t *testing.T, c *api.Connection, spaceName string, logs []string) []interface{} {
```

Comment:
Can we instead use the `...` syntax for logs:
```suggestion
func postSpaceLogs(t *testing.T, c *api.Connection, spaceName string, logs ...string) []interface{} {
```

Reply:
👍